stack-templates.json

```diff
@@ -1899,5 +1934,69 @@
+    "type": 2
+  },
+  {
+    "categories": [
+      "camunda",
+      "camunda tomcat"
+    ],
+    "description": "Camunda Platform 7.7.0 running on Tomcat",
+    "logo": "https://portainer-io-assets.sfo2.digitaloceanspaces.com/logos/registry.png",
+    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/camunda-camunda-bpm-platform-tomcat-7.7.0/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/camunda-camunda-bpm-platform-tomcat-7.7.0/docker-stack.yml</a>",
+    "platform": "linux",
+    "repository": {
+      "stackfile": "stacks/camunda-camunda-bpm-platform-tomcat-7.7.0/docker-stack.yml",
+      "url": "https://github.com/camunda/portainer-templates"
+    },
+    "title": "Camunda Platform 7.7.0 Tomcat",
+    "type": 2
+  },
+  {
+    "categories": [
+      "camunda",
+      "camunda tomcat"
+    ],
+    "description": "Camunda Platform 7.6.0 running on Tomcat",
+    "logo": "https://portainer-io-assets.sfo2.digitaloceanspaces.com/logos/registry.png",
+    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/camunda-camunda-bpm-platform-tomcat-7.6.0/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/camunda-camunda-bpm-platform-tomcat-7.6.0/docker-stack.yml</a>",
+    "platform": "linux",
+    "repository": {
+      "stackfile": "stacks/camunda-camunda-bpm-platform-tomcat-7.6.0/docker-stack.yml",
+      "url": "https://github.com/camunda/portainer-templates"
+    },
+    "title": "Camunda Platform 7.6.0 Tomcat",
+    "type": 2
+  },
+  {
+    "categories": [
+      "camunda",
+      "camunda tomcat"
+    ],
+    "description": "Camunda Platform 7.5.0 running on Tomcat",
+    "logo": "https://portainer-io-assets.sfo2.digitaloceanspaces.com/logos/registry.png",
+    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/camunda-camunda-bpm-platform-tomcat-7.5.0/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/camunda-camunda-bpm-platform-tomcat-7.5.0/docker-stack.yml</a>",
+    "platform": "linux",
+    "repository": {
+      "stackfile": "stacks/camunda-camunda-bpm-platform-tomcat-7.5.0/docker-stack.yml",
+      "url": "https://github.com/camunda/portainer-templates"
+    },
+    "title": "Camunda Platform 7.5.0 Tomcat",
+    "type": 2
   }
```

Comment:
🔧 I'd probably rather remove those entries from the `new-stack-templates.json` so they are not generated anymore.

Reply:
Done.